x/stakeibc/keeper/hooks.go

```diff
@@ -186,7 +193,7 @@ func (k Keeper) CreateDepositRecordsForEpoch(ctx sdk.Context, epochNumber int64)
 
 func (k Keeper) SetWithdrawalAddress(ctx sdk.Context) {
 	setWithdrawalAddresses := func(ctx sdk.Context, index int64, zoneInfo types.HostZone) error {
-		k.Logger(ctx).Info("\tsetting withdrawal addresses on host zones")
+		k.Logger(ctx).Info("\tsetting withdrawal address for index %d, zoneInfo %v", index, zoneInfo)
```

Comment:
this might just be something wrong my local setup, but I vaguely remember formatting like this not working for me, and I had to do `k.Logger(ctx).Info(fmt.Sprintf(...))`

Reply:
Done.